openslides/core/views.py

```diff
@@ -35,6 +36,14 @@ class IndexView(utils_views.View):
     to the custom staticfiles directory. See STATICFILES_DIRS in settings.py.
     """
 
+    @classmethod
```

Comment:
Sorry I don't understand. The IndexView only appears once in the ulrs.py so you can decorate it there. Or you can decorate the dispatch method of the class so that the decorater has effect on every view of this class.